src/librustdoc/html/markdown.rs

```diff
@@ -531,6 +530,7 @@ extern {
     fn hoedown_buffer_new(unit: libc::size_t) -> *mut hoedown_buffer;
     fn hoedown_buffer_puts(b: *mut hoedown_buffer, c: *const libc::c_char);
     fn hoedown_buffer_free(b: *mut hoedown_buffer);
+    fn hoedown_buffer_put(b: *mut hoedown_buffer, c: *const u8, len: libc::size_t);
```

Comment:
is `hoedown_buffer_puts` used elsewhere? This PR seems mostly to be replacing puts with put. If puts isn't, then we should also remove it